test/test_parse.py

```diff
@@ -0,0 +1,57 @@
+import unittest
+
+from psqlparse import parse
+from psqlparse.exceptions import PSqlParseError
+from psqlparse.nodes import WhereClause
+
+
+class TestParse(unittest.TestCase):
+
+    def test_select_all_no_where(self):
+        query = "SELECT * FROM my_table"
+        stmt = parse(query).pop()
+        self.assertEqual(stmt.type, "SelectStmt")
+        self.assertIsNone(stmt.where_clause)
+        target = stmt.target_list.targets.pop()
+        self.assertDictEqual({'A_Star': {}},
+                             target['val']['ColumnRef']['fields'][0])
+
+    def test_select_one_column_where(self):
+        query = "SELECT col1 FROM my_table WHERE my_attribute LIKE condition"
+        stmt = parse(query).pop()
+        self.assertEqual(stmt.type, "SelectStmt")
+        self.assertIsInstance(stmt.where_clause, WhereClause)
+        target = stmt.target_list.targets.pop()
+        self.assertDictEqual({'str': 'col1'},
+                             target['val']['ColumnRef']['fields'][0]['String'])
+        self.assertDictEqual({'str': '~~'},
+                             stmt.where_clause.obj['A_Expr']['name'][0]['String'])
+
+    def test_select_join(self):
+        query = "SELECT * FROM table_one JOIN table_two USING (common)"
+        stmt = parse(query).pop()
```

Comment:
Yep, it is. I think it is being raised by any join statement, should it be skipped it?